### DISCUSSION TITLE
Add timestamp to table for uploads and downloads

### DISCUSSION THREAD

**Discussion by deafwired** [nicotine-plus/nicotine-plus#2612] [Ideas]
I think it would be useful to add a timestamp in the table for downloads and uploads. It would also be nice to add a section in the settings that let you use [strftime](https://docs.python.org/3/library/time.html?highlight=strftime#time.strftime) to format it (%b-%d-%Y -> Aug-6-2023) individually for downloads and uploads, similar to the log timestamp format.

**Comment by slook** [comment]
> I think it would be useful

I agree. Please can you elaborate as to exactly why this would be useful to you?

**Reply by deafwired** [reply]
When I use nicotine+, I leave it in the background so that people can get my files, If I open nicotine+ later I don't know when they downloaded it and I think that would be nice to know. Also afaik there is no way to return the tables to chronological order after sorting them by something else.

**Reply by slook** [reply]
Since 3.3.0.dev5, If you click on the column header for a third time then it returns to the natural sort order.
